File: suma_promedio.py
import time

inicio = time.time() 

suma = 0
for i in range(10):
    valor = float(input("Ingresa un valor: "))
    suma = suma + valor

promedio = suma / 10

print("La suma total es:", suma)
print("El promedio es:", promedio)

fin = time.time() 

tiempo_total = fin - inicio
print("Tiempo de ejecución:", tiempo_total, "segundos")

File: suma_promedio.cpp
#include <stdio.h>

int main() {
    float suma = 0.0;
    float valor;

    for (int i = 0; i < 10; i++) {
        printf("Ingresa un número: ");
        scanf("%f", &valor); 
        suma = suma + valor;
    }

    float promedio = suma / 10.0;

    printf("La suma total es: %.2f\n", suma);
    printf("El promedio es: %.2f\n", promedio);

    return 0;
}